charts/mongodb/remedy.md

```diff
@@ -0,0 +1,90 @@
+## TODOs on MongoDB Critical Alerts
+
+### Database Alerts
+
+- #### MongoDBDown
+  - Describe the `MongoDB` CR, check the reason from conditions and try restarting the pods
+  - Check database pod logs
+  - Contact AppsCode team
+- #### MongodbReplicationLag
+  - Check if there is any network issue which is causing the replication lag.
+- #### MongodbTooManyConnections
+  - Increase `net.maxIncomingConnections` value using [MongoDBOpsRequest](https://kubedb.com/docs/latest/guides/mongodb/reconfigure/overview/)
+- #### MongodbVirtualMemoryUsage
+  - Increase storage and/or memory using [VerticalScaling OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/scaling/vertical-scaling/overview/) and [VolumeExpansion OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/volume-expansion/overview/)
```

Comment:
We can split this alert into two, like `MongoDBVirtualMemoryUsage` & `MongoDBStorageUsage` as referred opsReq is different.

Reply:
Virtual memory is the summation of storage and memory. I think, in this case, the user needs to decide whichever they want to increase.

Reply:
`MongoDBLowStorage` should be another alert. We need to add that, maybe in the future.